### ISSUE TITLE
Incorrect Label with Laravel 5.1

### ISSUE THREAD

**Issue by mwaa** [dwightwatson/bootstrap-form#34]
When using ``` BootForm::email('email') ```. Instead of the label been `email` its generated as `forms.email`. This is also true for text and the rest.

```html
<label for="email" class="control-label">forms.email</label>
```

**Comment by dwightwatson** [comment]
Whoops, looks like the translator changed a little bit with Laravel 5.1. I've pushed and tagged a fix (`1.0.4`).

**Comment by mwaa** [comment]
@dwightwatson Thanks for the quick update
